sage-ios/SAGE/SAGE/EditProfileController.swift

```diff
@@ -62,9 +63,9 @@ class EditProfileController: FormController {
             }
         }))
         
-        actionSheet.addAction(UIAlertAction(title: "Reset to Default", style: .Default, handler: { (alertAction) -> Void in
+        actionSheet.addAction(UIAlertAction(title: "Remove Photo", style: .Default, handler: { (alertAction) -> Void in
```

Comment:
add the action only if the user.imageURL != nil and make the style = .Destructive and put this at the top